### PR TITLE
GNNE-1904 Fix/quant test

### PR DESCRIPTION
**Pull request by Hejunchao6**

### CODE REVIEW

File: tests/kernels/test_dequantize.cpp [hunk 1]
```diff
@@ -81,46 +87,90 @@ TEST_P(DequantizeTest, dequantize) {
...
+        quant_param_t quant_param[] = {quantParam};
+        auto quant_param_ptr =
+            hrt::create(dt_int64, {1},
+                        {reinterpret_cast<gsl::byte *>(quant_param),
+                         sizeof(quant_param)},
+                        true, host_runtime_tensor::pool_cpu_only)
+                .expect("create tensor failed");
+
+        // expected
+        auto output1 = kernels::stackvm::dequantize(dt_float32, input.impl(),
```
Comment (xhuohai) [inline tests/kernels/test_dequantize.cpp:149]: 自己跟自己比没有意义

**Review by xhuohai — CHANGES_REQUESTED** [review]

File: tests/kernels/test_dequantize.cpp (hunk 1 above)
Comment (Hejunchao6) [reply tests/kernels/test_dequantize.cpp]: done

**Comment by xhuohai** [comment]
please bind jira ID with PR's title

**Comment by Hejunchao6** [comment]
> please bind jira ID with PR's title

done

**Review by xhuohai — APPROVED** [review]

**Pull request merged**